src/Postmark.Tests/ClientTemplateTests.cs

```diff
@@ -204,15 +204,17 @@ public async void ClientCanUseLayoutTemplatesWhenValidating()
         public async void ClientCanSendWithTemplate()
         {
             var template = await _client.CreateTemplateAsync("test template name", "test subject", "test html body");
-            var sendResult = await _client.SendEmailWithTemplateAsync(template.TemplateId, new { name = "Andrew" }, WRITE_TEST_SENDER_EMAIL_ADDRESS, WRITE_TEST_SENDER_EMAIL_ADDRESS, false);
+            var sendResult = await _client.SendEmailWithTemplateAsync(template.TemplateId, new { name = "Andrew" }, 
+                WRITE_TEST_SENDER_EMAIL_ADDRESS, WRITE_TEST_SENDER_EMAIL_ADDRESS, false, messageStream: PostmarkMessageBase.DefaultTransactionalStream);
```

Comment:
Further unit tests will be added when implementing the actual `MessageStream` API functions.